cloud_governance/policy/helpers/aws/aws_policy_operations.py

```diff
@@ -263,3 +270,25 @@ def get_network_out_kib_metric(self, resource_id: str, days: int = INSTANCE_IDLE
         average_network_out_bytes = self.__get_aggregation_metrics_value(metrics.get('MetricDataResults', []),
                                                                          aggregation='average')
         return round(average_network_out_bytes / TOTAL_BYTES_IN_KIB, DEFAULT_ROUND_DIGITS)
+
+    def __get_db_connection_status(self, resource_id: str, days: int = CLOUDWATCH_METRICS_AVAILABLE_DAYS):
+        start_date, end_date = Utils.get_start_and_end_datetime(days=days)
+        metrics = self._cloudwatch.get_metric_data(resource_id=resource_id, start_time=start_date, end_time=end_date,
+                                                   resource_type='DBInstanceIdentifier',
+                                                   metric_names={'DatabaseConnections': 'Count'},
+                                                   namespace='AWS/RDS', statistic='Maximum'
+                                                   )
+        total_connections = self.__get_aggregation_metrics_value(metrics.get('MetricDataResults', []),
+                                                                 aggregation='sum')
+        return total_connections
+
+    def is_database_idle(self, resource_id: str):
+        """
+        This method returns bool on verifying the database connections
+        :param resource_id:
+        :type resource_id:
+        :return:
+        :rtype:
+        """
+        total_connections = self.__get_db_connection_status(resource_id)
+        return total_connections <= 0
```

Comment:
Why total_connections <= 0
I think it should be: return total_connections

Reply:
Method name: _is_database_idle()_

Reply:
In idle case it should be <= 0

Reply:
Can it be -1 ?